matchms/Scores.py

```diff
@@ -1,16 +1,59 @@
-from typing import Callable
+from __future__ import annotations
 from typing import List
 from typing import Tuple
 import numpy
 from matchms.typing import QueriesType
 from matchms.typing import ReferencesType
+from matchms.typing import SimilarityFunction
 
 
 class Scores:
-    """An example docstring for a class definition."""
-    def __init__(self, references: ReferencesType, queries: QueriesType, similarity_function: Callable):
-        """An example docstring for a constructor."""
+    """Scores container
```

Comment:
Could we add an extra explanation beyond "Scores container" , which probably is too computational/technical to make sense for many mass spec users.
For instance: "A scores instance contains reference and query spectrums and allows calculating similarity scores between those spectrums." (maybe also mention querying possibility? )

Reply:
Rephrased